apps/console/app/routes/_shell.agent.$agentSlug.branches/route.tsx

```
@@ -0,0 +1,90 @@
+        });
+      } catch (error) {
+        return submission.reply({ formErrors: [ parseError(error).message ] });
+      }
+      
+      if (result.error?.status === 409) 
+        return submission.reply({ fieldErrors: { branchName: [String(result.error.value)] }});
+
+      break;
+
+    case "delete": 
+      const branchSlug = formData.get("branchSlug") as string;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)